arangod/Pregel/Worker/Handler.h

```diff
@@ -283,60 +130,9 @@ struct WorkerHandler : actor::HandlerBase<Runtime, WorkerState<V, E, M>> {
         arangodb::pregel::metrics::message::WorkerGssStarted{.threadsAdded =
                                                                  1});
```

Comment:
These lines should be deleted, this message is already sent inside the Computing state